exercises/02-publish-subscribe/after/Divergent.Customers/Program.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Configuration;
+using System.Linq;
+using System.ServiceProcess;
+using System.Threading.Tasks;
+
+namespace Divergent.Customers
+{
+    static class Program
+    {
+        public async static Task Main(string[] args)
+        {
+            var host = new Host(ConfigurationManager.ConnectionStrings[Host.EndpointName].ToString());
+
+            // pass this command line option to run as a windows service
+            if (args.Contains("--run-as-service"))
```

Comment:
Question : Why is this implementation so different from [our documentation](https://docs.particular.net/nservicebus/hosting/windows-service)?

Reply:
@dvdstelt I believe that documentation needs updating. This implementation is based on our [dotnet template](https://github.com/Particular/dotnetTemplates/tree/develop/src/Templates/NServiceBusWindowsService).